### PR TITLE
FMPy for bouncing_ball test

### PR DESCRIPTION
**Pull request by 0815Creeper**

### CODE REVIEW

**Comment by 0815Creeper** [comment]
if tests pass: ready to merge

**Comment by ThummeTo** [comment]
You can use the modelica reference FMUs with `loadFMU(model, "ModelicaReferenceFMUs", version)` if FMIZoo is loaded. No need to add a binary to the repo :-)

File: .github/workflows/TestLatest.yml [hunk 1]
```diff
@@ -16,11 +16,11 @@ jobs:
   test:
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: true
+      fail-fast: false # keep other os running if one fails
       matrix:
         julia-version: ['1']
         julia-arch: [x64]
-        os: [windows-latest] # ubuntu-latest, 
+        os: [windows-latest, ubuntu-latest] # ubuntu-latest,
```
Comment (ThummeTo) [inline .github/workflows/TestLatest.yml:23]: does this work? FMIExport only runs on windows for now

**Comment by ThummeTo** [comment]
closes #46

File: .github/workflows/TestLatest.yml (hunk 1 above)
Comment (0815Creeper) [reply .github/workflows/TestLatest.yml:23]: > does this work? FMIExport only runs on windows for now

yes it does. Thats the test-action beeing ubuntu-ready already, although FMIExport ist not ready yet.

**Comment by 0815Creeper** [comment]
> You can use the modelica reference FMUs with `loadFMU(model, "ModelicaReferenceFMUs", version)` if FMIZoo is loaded. No need to add a binary to the repo :-)

If you take a look at [test/bouncing_ball/bouncing_ball.jl](https://github.com/ThummeTo/FMIExport.jl/pull/56/files#diff-a500fa6bde27c32972316b6e4ad65098e9e8c85f3da18461c7beac77eff3c4f6) you will notice, that is exactly what is done for ubuntu. For Windows we want to test the generated FMU. As soon as it can by simulated using FMI, removing the .fmu file from the repo, it will switch to that. If you prefer modifying test/bouncing_ball/bouncing_ball.jl at that point, I will change this.

**Comment by ThummeTo** [comment]
> > You can use the modelica reference FMUs with `loadFMU(model, "ModelicaReferenceFMUs", version)` if FMIZoo is loaded. No need to add a binary to the repo :-)
> 
> If you take a look at [test/bouncing_ball/bouncing_ball.jl](https://github.com/ThummeTo/FMIExport.jl/pull/56/files#diff-a500fa6bde27c32972316b6e4ad65098e9e8c85f3da18461c7beac77eff3c4f6) you will notice, that is exactly what is done for ubuntu. For Windows we want to test the generated FMU. As soon as it can by simulated using FMI, removing the .fmu file from the repo, it will switch to that. If you prefer modifying test/bouncing_ball/bouncing_ball.jl at that point, I will change this.

I still don't see the point: why is it necessary to add a FMU binary to this repo? ModelicaReferenceFMUs exist for Linux and Windows as well. The idea behind FMIZoo is to prevent adding binaries for testing purpose.

**Comment by ThummeTo** [comment]
> What is the plan with this PR going forward?

We merge it as soon as all tests pass, thats the idea behind CI.
Question is how we can get there, it seems like we are using an updated version of windows std libraries, because `if (GetModuleFileName(hm, path, sizeof(path)) == 0)` errors in FMU_init.c. This needs to be fixed in FMIBuild (PR is open).

As soon as all tests pass, we can merge this PR.

**Pull request closed without merging**